uaclient/messages.py

```diff
@@ -823,8 +833,8 @@ class TxtColor:
     bold=TxtColor.BOLD, end_bold=TxtColor.ENDC
 )
 REALTIME_PRE_DISABLE_PROMPT = """\
-This will disable Ubuntu Pro updates to the Real-time kernel on this machine.
-The Real-time kernel will remain installed.\
+This will uninstall the Real-time kernel package.
+You are responsible for configuring a kernel to boot after this action.
```

Comment:
Should we better alert that the realtime-kernel will not indeed be uninstalled ? Just to highlight to users that they really to manually change the kernel configuration

Reply:
Agreed; imho this message could be either more explanatory or "scarier".